CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-api-python3#89](https://github.com/cyberark/conjur-api-python3/issues/89)
 
 ### Changed
+- CLI command UX has been improved according to UX guidelines
```

Comment:
@orenbm how do we feel about this changelog message?

Reply:
I think it's good. the link below has the needed explanations